index.ts

```diff
@@ -83,6 +97,22 @@ const DeleteCacheArgsSchema = z.object({
   name: z.string().describe("The name of the cache to delete"),
 });
 
+const PublishTopicArgsSchema = z.object({
+  cacheName: z
+    .string()
+    .describe(
+      "The name of the cache to set the value in. Uses the default cache name if not provided."
+    )
+    .optional(),
+  topicName: z
+    .string()
+    .describe(
+      "The name of the topic to publish the value to. Uses the default topic name if not provided."
+    )
+    .optional(),
+  value: z.string().describe("The value to set in the cache"),
```

Comment:
The value to publish

Reply:
great catch! fixed in https://github.com/momentohq/mcp-momento/pull/9/commits/5515002507f3a11100c832251c11aae74d493102